src/main/resources/templates/doctor-profile.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <link href="https://fonts.googleapis.com/css2?family=Gill+Sans&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../style.css" type="text/css">
+    <title>Doctor Portal</title>
+</head>
+<body>
+
+<div class="header">
+
+    <nav>
+        <ul>
+            <li><a href="../">Home</a></li>
+            <li><a href="../">Patient Portal</a></li>
+            <li><a th:href="@{'/doctorView/'}">Doctor Portal</a></li>
+            <li><a href="../">Pharmacy</a></li>
+            <li><a href="../">Scheduling</a></li>
+        </ul>
+    </nav>
+</div>
+
+<div class="title-header">
+    <p>Doctor Portal -  <span th:if="${doctor.isPresent()}" th:text="${doctor.get().name}"></span></p>
+</div>
+
+<div class="content">
+
+    <div class="patient-info">
+        <p><b>Profession:</b></p>
+        <p><span th:if="${doctor.isPresent()}" th:text="${doctor.get().profession}"></span></p>
+    </div>
+
+    <div class="patient-info">
+        <p><b>Next Appointment:</b></p>
```

Comment:
Opinion:
If this is from the patient perspective, we do not want to display the doctors appointments and what patient the appointment is with to all patients.

We also should not display the doctor's patient list.
We should keep this page simple only displaying the doctor name and profession.